hkm/management/commands/add_admin_user.py

```diff
@@ -10,14 +13,28 @@ def add_arguments(self, parser):
             "-u", "--username", type=str, help="Username", default="admin"
         )
         parser.add_argument(
-            "-p", "--password", type=str, help="Password", default="admin"
+            "-p", "--password", type=str, help="Password", default=""
         )
         parser.add_argument(
             "-e", "--email", type=str, help="Email", default="admin@example.com"
         )
 
     def handle(self, *args, **kwargs):
         if not get_user_model().objects.filter(username=kwargs["username"]).count():
+            password = kwargs["password"]
+            if not password:
+                password = self.random_string(20)
```

Comment:
Just FYI, there seems to be [a built-in way](https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.models.BaseUserManager.make_random_password) to accomplish something similar:

```python
from django.contrib.auth import get_user_model
User = get_user_model()
password = User.objects.make_random_password(20)
```

Reply:
Thanks for the tip, I was unaware of this + my question on #kuva-aok-development didn't get any responses.

I'll remove my custom stuff and replace it with your suggestion.

Reply:
I've pushed the change now.